src/services/production.ts

```diff
@@ -112,7 +111,6 @@ export const services: ServiceDefinition[] = [
   [$.bootstrap, {
     service: useBootstrap,
     arguments: [
-      $.env,
```

Comment:
@johncowen One think to look out for I just noticed: This will no longer tell you if the arguments are wrong for the service construction. Previously, we would get a TypeScript error when trying to pass in env here while removing the argument from the `useBootstrap` function. This doesn’t happen here now.

Reply:
Oh right, I will take look at that! Thanks! 👍